homeassistant/components/tomorrowio/weather.py

```diff
@@ -221,19 +251,21 @@ def forecast(self):
             condition = values.get(TMRW_ATTR_CONDITION)
             precipitation = values.get(TMRW_ATTR_PRECIPITATION)
             precipitation_probability = values.get(TMRW_ATTR_PRECIPITATION_PROBABILITY)
+            if precipitation_probability is not None:
```

Comment:
Just in case we get bad data, maybe this would be better.

`if isinstance(precipitation_probability, Number):`